doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -566,78 +566,96 @@ The query returns all the entities that match all the matching conditions (AND l
 
 There are two kinds of statements: *unary statements* and *binary statements*.
 
-Binary statements are composed by an attribute name (e.g. "temperature"), an operator and a value
+Binary statements are composed by an attribute path (e.g. "temperature"), an operator and a value
 (whose format depends on the operator), e.g.:
 
 ```
 temperature==50
 temperature<=20
 ```
 
+The attribute path is used to select the *element under evaluation* for the statement. Its general
+syntax consists of an attribute name optionally followed by a *navigation path* using the `.`
+character as delimiter, e.g. "temperature.accuracy.average" refers to the navigation path "accuracy.average"
+within the "temperature" attribute.
+
+How the navigation path is interpreted depends on where the Simple Query Language statement is used:
+
+* If used in a `q` query, then the navigation path refers to a sub-key of the attribute value
+  (assuming that the attribute value is a JSON object). If no navigation path is provided (i.e.
+  the attribute path doesn't have any `.`) then the attribute itself is taken into account.
```

Comment:
After 76530a8, this has been deeply rewritten, so please consider the original comment as deprecated and re-evaluate it in the new text.